eduiddashboard/views/nins.py

```diff
@@ -512,7 +512,7 @@ def resendcode(self):
         verified_nins = user.get_nins()
 
         if len(verified_nins) > 0:
-            message = _("You already have a verified national identity number")
+            message = _("You already have a confirmed national identity number")
```

Comment:
I was planing on commiting those directly to master, however since it will be done right after merge I have now included the updated files as well.